src/backend/storeManagers/hyperplay/games.ts

```diff
@@ -1655,6 +1658,82 @@ export async function downloadGameIpdtManifest(
   }
 }
 
+async function checkIfPatchingIsFaster(
+  oldManifestPath: string,
+  newManifestPath: string,
+  gameInfo: GameInfo
+) {
+  // read manifests
+  const oldManifestJson = JSON.parse(readFileSync(oldManifestPath).toString())
+  const newManifestJson = JSON.parse(readFileSync(newManifestPath).toString())
+
+  // compare manifests
+
+  const { compareManifests } = await import('@hyperplay/patcher')
+  const blockSizeForEstimateInKB = 140
```

Comment:
so, on the client we are using 512KB and I think all blocks are being generated around that now right  @jiyuu-jin  ?

Reply:
@flavioislima see the comment block on this pr https://github.com/HyperPlay-Gaming/hyperplay-dev/pull/533/files

Reply:
Ah ok, I see now 👍🏽